lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -1735,6 +1735,11 @@ ThreadContext::ProbeStack(size_t size, Js::ScriptContext *scriptContext, PVOID r
 {
     this->ProbeStackNoDispose(size, scriptContext, returnAddress);
 
+    if (PlatformAgnostic::PerfTrace::mapsRequested)
+    {
+        PlatformAgnostic::PerfTrace::WritePerfMap();
```

Comment:
On Windows since the implementation is empty, can you look at the code generated for ProbeStack and confirm that this if-check is optimized away?